crates/sui-indexer/src/apis/indexer_api.rs

```diff
@@ -215,11 +217,11 @@ impl<S: IndexerStore> IndexerApi<S> {
         })
     }
 
-    async fn get_owned_objects_interal(
```

Comment:
Can we return error when the filter is not `address owner` and add a `MUSTFIX` here? I want to avoid accidentally enabling this endpoint because we forget this is not fully ready yet